Dockerfile

```diff
@@ -53,6 +53,7 @@ COPY pom.xml /build/pom.xml
 WORKDIR /build
 
 RUN mvn -B -e -T 1C -DskipTests=true -DfailIfNoTests=false -Dtest=false clean package -Pdist
+RUN mvn dependency:copy -Dartifact=org.postgresql:postgresql:42.2.16:jar -DoutputDirectory=/build -Dmdep.stripVersion
```

Comment:
@bentito do we have a preference for how we get this artifact? I think ideally, I would prefer grabbing it from the base image or just wget/curl from the jdbc driver artifacts from the download page.

Reply:
Not opposed- but we have maven in there and this image takes close to an hour to build so since we all ready have it grabbing postgres, might want to think about pure time/ longveity trade offs

Reply:
This is actually a great way to get it just because maven repos seldom go away for a given versiion. So I'd say if there is an external dep to a Maven built project, get it with Maven too.